models.py

```diff
@@ -42,3 +38,4 @@ def save(self, *args, **kwargs):
         if self.winner_handle != self.loser_handle:
             return super(Match, self).save(*args, **kwargs)
         raise IntegrityError('Winner cannot be the same as loser')
+
```

Comment:
destroy

Reply:
Huh?

Reply:
no empty line at end of file pls

Reply:
Grr, apparently kakoune doesn't display the trailing newline so I've been putting extras on all my work.